doc/branches.md

```diff
@@ -1,39 +1,82 @@
-| crvcoincs |   information about a cluster of CRV coincidence triplets| [see CrvHitInfoReco.hh](../inc/CrvHitInfoReco.hh)
-| crvcoincsmc |   information about the MC track which most likely caused the CRV coincidence triplets| [see CrvHitInfoMC.hh](../inc/CrvHitInfoMC.hh)
-| crvcoincsmcplane |   information about the point where the MC trajectory crosses the xz plane of CRV-T| [see CrvPlaneInfoMC.hh](../inc/CrvPlaneInfoMC.hh)
-| trkqual |   the output of a multi-variate analysis (MVA)| [see MVAResultInfo.hh](../inc/MVAResultInfo.hh)
-## Vector-of-Vector Branches
-
-| branch | explanation | leaf information |
-|--------|-------------|------------------|
-| trksegs |   track fit results at particular surfaces| [see TrkSegInfo.hh](../inc/TrkSegInfo.hh)
-| trksegpars_lh |   LoopHelix track parameters at each trkseg surface (used for looping tracks)| [see LoopHelixInfo.hh](../inc/LoopHelixInfo.hh)
-| trksegpars_ch |   CentralHelix track parameters at each trkseg surface (used for field-on cosmic tracks)| [see CentralHelixInfo.hh](../inc/CentralHelixInfo.hh)
-| trksegpars_kl |   KinematicLine track parameters for each trkseg surface (used for field-off cosmics)| [see KinematicLineInfo.hh](../inc/KinematicLineInfo.hh)
-| trkmcsim |   information about SimParticles in genealogy| [see SimInfo.hh](../inc/SimInfo.hh)
-| trkhits |   information about the straw hits assigned to a track| [see TrkStrawHitInfo.hh](../inc/TrkStrawHitInfo.hh)
-| trkhitsmc |   MC-truth information of straw hits assigned to a track| [see TrkStrawHitInfoMC.hh](../inc/TrkStrawHitInfoMC.hh)
-| trkmats |   information on the straw materials used in the Kalman fit| [see TrkStrawMatInfo.hh](../inc/TrkStrawMatInfo.hh)
-| trkmcsci |   information about StepPointMCs| [see MCStepInfo.hh](../inc/MCStepInfo.hh)
-| trkmcssi |   information that summarizes all StepPointMCs in a given location| [see MCStepSummaryInfo.hh](../inc/MCStepSummaryInfo.hh)
-| trksegsmc |   MC-truth information of the SurfaceSteps taken by a particle through passive elements, or virtual detectors| [see SurfaceStepInfo.hh](../inc/SurfaceStepInfo.hh)
+## Event Branches
+
+These branches contain one element per event.
+
+| branch | structure | explanation | leaf information |
+|--------|-----------|-------------|------------------|
+| evtinfo |  Single objects branch |   event-level information| [see EventInfo.hh](../inc/EventInfo.hh)
+| evtinfomc |  Single objects branch |   MC-truth event-level information| [see EventInfoMC.hh](../inc/EventInfoMC.hh)
+| hitcount |  Single objects branch |   counts of different hit types in an event| [see HitCount.hh](../inc/HitCount.hh)
+| tcnt |  Single objects branch |   counts track types and track-related quantities in an event (MARKED FOR REMOVAL)| [see TrkCount.hh](../inc/TrkCount.hh)
+## Track Branches
+
+These branches contain 4 elements per event corresponding to different Kalman fit hypotheses:
```

Comment:
Technically, it could be less than 4 elements if e.g. the upstream fits completely fail or are not done. It could also be more than 4 elements if there are e.g. two real downstream tracks to fit. I think we want to make the point that there will be multiple tracks in each event and they could be the same track just with different fit hypotheses...